src/core/dx-template.ts

```diff
@@ -21,16 +21,23 @@ function createDxTemplate(
     keyFn?: (data: any) => string
 ): IDxTemplate {
 
-    const renderedContainers: HTMLElement[] = [];
+    const renderedModels: Array<{ model: any, templateId: string }> = [];
```

Comment:
Let's clear the `renderedModels` array on template removal to avoid memory leaks.

Reply:
Done